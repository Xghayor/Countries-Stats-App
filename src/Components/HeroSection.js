import React from 'react';
import './styles/HeroSection.css';
import maps from './assets/maps.png';

const HeroSection = () => {
  return (
    <div className="hero">
      <div className="hero-image">
        <img src={maps} alt="Globe" />
      </div>
      <div className="hero-content">
      <h1 className="hero-title">Worldwide Insights</h1>
      </div>
    </div>
  );
};

export default HeroSection;
