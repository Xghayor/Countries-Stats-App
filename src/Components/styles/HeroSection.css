
.hero {
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: rgba(46, 128, 228, 0.772);
  padding: 20px; 
}


.hero-image {
  flex: 1; 
  padding: 10px; 
}


.hero-image img {
  max-width: 80%; 
  height: auto;
  opacity: 0.8;
  margin-left: 10%;
}

.hero-content {
  flex: 1; 
  padding: 20px; 
}


.hero-title {
  font-size: 36px; 
  font-weight: bold; 
  color: white; 
  text-transform: uppercase; 
  text-align: center; 
  margin-right: 300px;
  letter-spacing: 2px;
  text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.3);
}

