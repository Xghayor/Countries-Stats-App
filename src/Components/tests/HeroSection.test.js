import React from 'react';
import { render } from '@testing-library/react';
import HeroSection from '../HeroSection';

test('HeroSection component renders correctly', () => {
  const { container } = render(<HeroSection />);
  expect(container).toMatchSnapshot();
});
