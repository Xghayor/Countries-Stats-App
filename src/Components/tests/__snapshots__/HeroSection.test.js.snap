// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroSection component renders correctly 1`] = `
<div>
  <div
    class="hero"
  >
    <div
      class="hero-image"
    >
      <img
        alt="Globe"
        src="maps.png"
      />
    </div>
    <div
      class="hero-content"
    >
      <h1
        class="hero-title"
      >
        Worldwide Insights
      </h1>
    </div>
  </div>
</div>
`;
